webapp/components/survey/CategoryDetails/store/actions/category/useInit.js

```diff
@@ -33,8 +34,10 @@ const _fetchCategory = async ({ surveyId, categoryUuid }) => {
 export const useInit = ({ setState }) => {
   const dispatch = useDispatch()
   const surveyId = useSurveyId()
+  const fetchLevelItems = useFetchLevelItems({ setState })
 
-  return useCallback(async ({ categoryUuid, onCategoryCreated }) => {
+  return useCallback(async ({ state, onCategoryCreated }) => {
+    const categoryUuid = State.getCategoryUuid(state)
     let category = null
 
     if (A.isEmpty(categoryUuid)) {
```

Comment:
can categoryUuid be empty? maybe not?

Reply:
this needs to be changed when the ButtonMetaItemAdd will be used to create the category